Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -2521,7 +3172,7 @@ final class SafeDIToolTests: XCTestCase {
         }
     }
 
-    func test_run_onCodeWithUnfulfillableRenamedReceivedPropertyType_throwsError() async {
```

Comment:
I made the tests refer to `Aliased` rather than `Renamed` for consistency. We still talk about renaming in the README, but that's fine.

I'm also open to feedback that the `case` should be `case renamed` rather than `case alias`